### PR TITLE
Additional AVS entries for AVS[16-19]

### PR DESCRIPTION
**Pull request by michielvanschaik**
# Overview/Summary

Additional AVS entries for AVS[16-19]

## This PR fixes/adds/changes/removes

1. Add AVS 16-19 to address availability, networking, disaster recovery, and identity and access management
2. Updated scripts where no ARG is available

## As part of this Pull Request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by ejhenry — COMMENTED** [review]
Thanks @michielvanschaik. For recommendations 17, 18, 19 can you update the recommendation category to use one of the options listed in the contribution guide?

https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#recommendation-categories

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 1]
```diff
@@ -14,15 +14,26 @@ The presented resiliency recommendations in this guidance include Azure VMware S
...
+|[AVS-12 Deploy two Azure VMware Solution private clouds in different regions for geographical disaster recovery](#avs-12---deploy-two-azure-vmware-solution-private-clouds-in-different-regions-for-geographical-disaster-recovery) | Disaster Recovery | High | Preview | No |
+|[AVS-13 Use the Interconnect feature to connect private clouds in different availability zones](#avs-13---use-the-interconnect-feature-to-connect-private-clouds-in-different-availability-zones) | Storage | High | Preview | No |
+|[AVS-14 Use key autorotation for vSAN datastore customer-managed keys](#avs-14---use-key-autorotation-for-vsan-datastore-customer-managed-keys) | Storage | High | Preview | No |
+|[AVS-15 Configure LDAPS Identity integration with two sources for NSX-T and vCenter management consoles](#avs-15---configure-ldaps-identity-integration-with-two-sources-for-nsx-t-and-vcenter-management-consoles) | Storage | High | Preview | No |
+|[AVS-16 Use Network Extension High Availability](#avs-16---use-network-extension-high-availability) | Availability | High | Preview | No |
+|[AVS-17 HCX Network Extension](#avs-17---hcx-network-extension) | Configuration | High | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:33]:
```suggestion
|[AVS-17 HCX Network Extension](#avs-17---hcx-network-extension) | Networking | High | Preview | No |
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 2]
```diff
@@ -14,15 +14,26 @@ The presented resiliency recommendations in this guidance include Azure VMware S
...
+|[AVS-18 Use multiple DNS servers per private FQDN zone](#avs-18---use-multiple-dns-servers-per-private-fqdn-zone) | Configuration | High | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:34]:
```suggestion
|[AVS-18 Use multiple DNS servers per private FQDN zone](#avs-18---use-multiple-dns-servers-per-private-fqdn-zone) | Networking | High | Preview | No |
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 3]
```diff
@@ -14,15 +14,26 @@ The presented resiliency recommendations in this guidance include Azure VMware S
...
+|[AVS-19 Verify vSAN FTT configuration aligns with the cluster size](#avs-19---verify-vsan-ftt-configuration-aligns-with-the-cluster-size) | Applications | High | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:35]:
```suggestion
|[AVS-19 Verify vSAN FTT configuration aligns with the cluster size](#avs-19---verify-vsan-ftt-configuration-aligns-with-the-cluster-size) | Application Resilience | High | Preview | No |
```

**Review by ejhenry — APPROVED** [review]
LGTM

**Review by Dost2010 — APPROVED** [review]

**Pull request merged**